pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -839,8 +839,6 @@ func mainEventLoop(zedagentCtx *zedagentContext, stillRunning *time.Ticker) {
 	getconfigCtx := zedagentCtx.getconfigCtx
 	dnsCtx := zedagentCtx.dnsCtx
 
-	hwInfoTiker := time.NewTicker(3 * time.Hour)
```

Comment:
It was 3 hours, and now the default is 12 hours. It can be a breaking change. It should be explicitly described at least. Or I would even suggest using the old defaults.